res/js/index.js

```diff
@@ -247,4 +247,28 @@ function negBuffer() {
 /** Helpers */
 function getById(id) {
 	return document.getElementById(id);
-}
\ No newline at end of file
+}
+
+ // ** For the timer */
+
+document.getElementById('timer').innerHTML = 20 + ":" + 05;
+startTimer();
+
+function startTimer() {
+  var presentTime = document.getElementById('timer').innerHTML;
+  var timeArray = presentTime.split(/[:]+/);
+  var m = timeArray[0];
+  var s = checkSecond((timeArray[1] - 1));
+  if(s==59){m=m-1}
+  //if(m<0){alert('timer completed')}
```

Comment:
This is a great start!

It looks like there is still some work to do in terms of deciding what to do when the timer is finished. I think for now, just a simple alert telling the user that the time is up is definitely good enough. 

With that said, I think that a possible way of finishing the implementation could be to:
  - Uncomment the `if` statement that alerts when the timer is completed, and set the time display in the HTML to be zeros
  - Add an `else` statement for all the cases where the timer is not yet completed, and re-set the timer in those cases.

Maybe that would look something like:
`
if (m<0) {
  document.getElementById('timer').innerHTML =
    "0" + ":" + "0";
  alert('timer completed');
}
else {
  document.getElementById('timer').innerHTML =
    m + ":" + s;
  setTimeout(startTimer, 1000);
}
`